### PR TITLE
mythic 0.4.2 (new cask)

### PR DESCRIPTION
**Pull request by mattmaniak**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
It was refused before in alpha version 0.3.1 (PR #178921) because of lack of a notarization. [Mythic bundle is notarized since 0.4.1](https://github.com/MythicApp/Mythic/issues/51#issuecomment-2464496744) so it's no longer a case.
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/m/mythic.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  version "0.4.2"
+  sha256 "13f10cedb4d410ec6e9b939df6ab96726c6e4aa9bde777cc1bfa862bb080b775"
+
+  url "https://dl.getmythic.app/sparkle-temp/Mythic-#{version}.zip"
+  name "Mythic"
+  desc "Unique open-source game launcher with the ability to run Windows games"
+  homepage "https://getmythic.app/"
+
+  livecheck do
+    url "https://github.com/MythicApp/Mythic/releases"
```
Comment (bevanjkay) [inline Casks/m/mythic.rb:11]:
```suggestion
    url "https://dl.getmythic.app/sparkle-temp/appcast.xml"
    strategy :sparkle, &:short_version
```

Comment (mattmaniak) [reply Casks/m/mythic.rb]: Thanks, fixed. I didn't know which strategy to use when using xml as a version provider so I have used GitHub.

File: Casks/m/mythic.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+cask "mythic" do
+  version "0.4.2"
+  sha256 "13f10cedb4d410ec6e9b939df6ab96726c6e4aa9bde777cc1bfa862bb080b775"
+
+  url "https://dl.getmythic.app/sparkle-temp/Mythic-#{version}.zip"
+  name "Mythic"
+  desc "Unique open-source game launcher with the ability to run Windows games"
```
Comment (bevanjkay) [inline Casks/m/mythic.rb:7]:
```suggestion
  desc "Game launcher with the ability to run Windows games"

**Review by bevanjkay — APPROVED** [review]

**Comment by khipp** [comment]
Thank you for your first contribution to `homebrew-cask`, @mattmaniak! 🎉

**Review by khipp — APPROVED** [review]

**Comment by mattmaniak** [comment]
> Thank you for your first contribution to `homebrew-cask`, @mattmaniak! 🎉

With struggles but I am happy that I won't drag-and-drop Mythic manually anymore :)

**Pull request merged**